src/functions/welcome/welcome.go

```diff
@@ -0,0 +1,66 @@
+package welcome
+
+import (
+	"encoding/json"
+	"fmt"
+	"html"
+	"net/http"
+	"strings"
+	"os"
+
+	"github.com/bwmarrin/discordgo"
+)
+
+// Data struct from Pubsub
+type PubSubMessage struct {
+	Data []byte `json:"data"`
+}
+
+type Payload struct {
+	UserId  string `json:"user_id"`
+	Username string `json:"username"`
+	ServerId string `json:"server_id"`
+	ServerName string `json:"server_name"`
+	AvatarUrl string `json:"avatar_url"`
+	ChannelId string `json:"channel_id"`
```

Comment:
Install the Go extension in VsCode to get code formatting on save:
https://marketplace.visualstudio.com/items?itemName=golang.Go

Reply:
ok thanks